hexa/pipelines/schema.py

```diff
@@ -378,6 +386,48 @@ def resolve_upload_pipeline(_, info, **kwargs):
         return {"success": False, "errors": [str(e)]}
 
 
+@pipelines_mutations.field("deletePipelineVersion")
+def resolve_delete_pipeline_version(_, info, **kwargs):
+    request: HttpRequest = info.context["request"]
+    input = kwargs["input"]
+    try:
+        pipeline = Pipeline.objects.filter_for_user(request.user).get(
+            id=input["pipelineId"]
+        )
+        pipeline_version = PipelineVersion.objects.get(id=input["versionId"])
+
+        if pipeline.versions.all().count() == 1:
+            return {"success": False, "errors": ["PERMISSION_DENIED"]}
+
+        if not request.user.has_perm("pipelines.delete_pipeline_version", pipeline):
+            return {"success": False, "errors": ["PERMISSION_DENIED"]}
+
+        # Only workspace admins can delete pipeline(s) version(s) created by others
+        if (
```

Comment:
Small detail: this could be done in the permissions module too.

Reply:
Oh yes, you're right.

Reply:
But I'm afraid that  if It will affect the check for delete_pipeline_version.
 ```python
@pipeline_permissions.field("deleteVersion")
def resolve_pipeline_permissions_delete_version(pipeline: Pipeline, info, **kwargs):
    request = info.context["request"]
    return request.user.is_authenticated and request.user.has_perm(
        "pipelines.delete_pipeline_version", pipeline
    )
```